app/components/myLibrary/libraryView.js

```diff
@@ -33,23 +28,46 @@ export default class LibraryView extends Component {
     this.state = {
       dataSource: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
       onPressedBookIndex: undefined,
-      onPressedColumn: undefined,
-      initialBookColor: [],
       fadeAnimRow: new Animated.Value(0),
       offset: 0,
-      direction: undefined
+      direction: undefined,
+      pan: new Animated.ValueXY()
     };
-    for (let i = 0; i < this.state.dataSource.length; i = i + 1) {
-      this.state.initialBookColor.push(generateRandomColor());
-    }
+    this.panResponder = PanResponder.create({
```

Comment:
this panresponder is not being used. I will either delete this or update it to use it